src/mpid/ch4/shm/posix/eager/iqueue/iqueue_send.h

```diff
@@ -24,35 +24,35 @@ MPL_STATIC_INLINE_PREFIX size_t MPIDI_POSIX_eager_buf_limit(void)
  * and retry.
  *
  * grank   - The global rank (the rank in MPI_COMM_WORLD) of the receiving process.
+ * buf, count, datatype - Data buffer and signature for the send buffer
+ * am_hdr, am_hdr_sz    - am header
+ * need_packing         - indicating of packing is needed for the send buffer, avoiding check it
+ *                        each time
  * msg_hdr - The header of the message to be sent. This can be NULL if there is no header to be sent
  *           (such as if the header was sent in a previous chunk).
- * iov     - The array of iovec entries to be sent.
- * iov_num - The number of entries in the iovec array.
+ * data_sz_left, offset - bytes left to be sent and current offset
```

Comment:
Why does it need `data_sz_left`?

Reply:
It need this to figure out much data need to be send, especially for the last segment where the size_sz_left would be smaller than cell size.